scipy/stats/tests/test_distributions.py

```diff
@@ -2922,6 +2922,22 @@ def test_sf_isf(self):
         xx = stats.invgamma.isf(y, 1)
         assert_allclose(x, xx, rtol=1.0)
 
+    @pytest.mark.parametrize("a, ref",
+                            [(100000000.0, -26.21208257605721),
+                            (10000000000.0, -33.11983786163935),
+                            (1e+20, -67.6586142566167),
+                            (1e+75, -257.6218844286255),
+                            (1e+100, -343.9688254159022)])
+    def test_large_entropy(self, a, ref):
+        # The reference values were calculated with mpmath:
+        # from mpmath import mp
+        # mp.dps = 500
+
+        # def invgamma_entropy(a):
+        #     h = a + mp.loggamma(a) - (mp.one + a) * mp.digamma(a)
```

Comment:
```suggestion
        # def invgamma_entropy(a):
        #     a = mp.mpf(a)
        #     h = a + mp.loggamma(a) - (mp.one + a) * mp.digamma(a)
```
```suggestion
        # def invgamma_entropy(a):
        #     h = a + mp.loggamma(a) - (mp.one + a) * mp.digamma(a)
```

Does this change the result?

Reply:
Doesn't seem to, no.

Reply:
Let's still add it so that future contributors use mpmath the proper way.